cmd/mattermost/commands/server.go

```diff
@@ -111,6 +111,9 @@ func runServer(configStore *config.Store, usedPlatform bool, interruptChan chan
 		return serverErr
 	}
 
+	fakeApp := app.New(app.ServerConnector(server))
+	fakeApp.InitServer()
```

Comment:
Should we just call `fakeApp.registerAppClusterMessageHandlers()` here to be as minimal as possible?

Along with a `TODO:` to remove this in future.

Reply:
Agreed on TODO, but there was another problem where the cluster was not starting schedulers since it's also was waiting the first request.

Reply:
Resolved offline.